README.md

```diff
@@ -1 +1,37 @@
 # Skeleton
+
+Create new apps quickly!
+
+## Installation
+
+Install this gem as follow:
+
+    $ gem install datacrafts-io-skeleton
+
+## Usage
+
+### New Rails API application without frontend part.
+
+    $ datacrafts-io-skeleton create APP_NAME
+    $ datacrafts-io-skeleton build APP_NAME
+    $ datacrafts-io-skeleton new APP_NAME
+
+## Development
+
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```

Comment:
Is it possible to run setup showing output and navigate to directory automatically with an appropriate notice?

Reply:
Not relevant, got it now